### ISSUE TITLE
Release w/ updated feature download URL?

### ISSUE THREAD

**Issue by willirath** [SciTools/cartopy#1849]
Would it be realistic to have a quick release containing #1837 and #1833?

As the old URL (naciscdn.org) seems completely unresponsive now (from my machines, even `$ host naciscdn.org` fails), a quick patch release would be very helpful for anyone using cartopy in settings without a persistent home dir (CI, some container setups) or for first-time users of cartopy who don't have a cached version of the features.

**Comment by markveillette** [comment]
I second this request.

**Comment by dnerini** [comment]
yes please, running in the same issue: a new release on PyPI would be very helpful in order to use the new download url.

**Comment by rcomer** [comment]
Thanks to #1837, the download script should work regardless of Cartopy version.  So you can [download it from master](https://raw.githubusercontent.com/SciTools/cartopy/master/tools/cartopy_feature_download.py) and use it to populate your shapefile cache.

Also note that [the latest release of Cartopy on conda-forge has been patched](https://github.com/conda-forge/cartopy-feedstock/pull/116), so if you use `conda`, you can update that way.

**Comment by willirath** [comment]
> Also note that [the latest release of Cartopy on conda-forge has been patched](https://github.com/conda-forge/cartopy-feedstock/pull/116), so if you use `conda`, you can update that way.

Ah, that's very helpful! Thanks!

**Comment by DanJonesOcean** [comment]
May I ask a potentially basic question about this? 

@willirath has been helping me with this issue over the past couple of days via twitter. But I'm still having problems. Let me tell you what I've tried. I opened up a Pangeo instance and used this command in the terminal:

`conda update -c conda-forge cartopy`

This started an update download and install, which was successful. I then opened a Jupyter notebook, which had been working a couple months ago when I last used it, and tried to make some cartopy plots. It didn't crash, although the error message was different than before. I'll post some of it here (the full stack track is very massive. I can share it if needed.)

```
/srv/conda/envs/notebook/lib/python3.8/urllib/request.py in https_open(self, req)
   1391 
   1392         def https_open(self, req):
-> 1393             return self.do_open(http.client.HTTPSConnection, req,
   1394                 context=self._context, check_hostname=self._check_hostname)
   1395 

/srv/conda/envs/notebook/lib/python3.8/urllib/request.py in do_open(self, http_class, req, **http_conn_args)
   1351                           encode_chunked=req.has_header('Transfer-encoding'))
   1352             except OSError as err: # timeout error
-> 1353                 raise URLError(err)
   1354             r = h.getresponse()
   1355         except:

URLError: <urlopen error [Errno -3] Temporary failure in name resolution>
```

I can also take this comment elsewhere if this isn't the right place for it. Thanks in advance for any help you can provide!

**Comment by rcomer** [comment]
I don’t know anything about Pangeo, but https://github.com/SciTools/cartopy/pull/1833#issuecomment-916086677 suggests it is necessary to specify `cartopy=0.19.0.post1` for the `conda install` on some platforms.  Probably worth a try.

**Comment by dopplershift** [comment]
@DanJonesOcean The important part there is to see what URL is being used. Can you tell if an updated Cartopy is actually being used by the jupyter notebook server? You could also type `%debug` in the notebook and poke around to see what the url is.

**Comment by andhuang-CLGX** [comment]
For those struggling in getting `conda install -c conda-forge cartopy=0.19.0.post1` to work, this is what I did to pre-populate the cached shapefiles:
```
git clone https://github.com/SciTools/cartopy.git
cd cartopy
cd tools
python3 cartopy_feature_download.py --help
python3 cartopy_feature_download.py cultural

# manually download and unzip states / lakes since it doesn't seem to be a group, even with `cultural-extra`
cd ~/.local/share/cartopy/shapefiles/natural_earth/cultural/
wget https://naturalearth.s3.amazonaws.com/110m_cultural/ne_110m_admin_1_states_provinces_lakes.zip 
unzip ne_110m_admin_1_states_provinces_lakes.zip
rm ne_110m_admin_1_states_provinces_lakes.zip
```

**Comment by greglucas** [comment]
We just released v0.20.0, which includes the new URLs.
